pydantic_ai_slim/pydantic_ai/_pydantic.py

```diff
@@ -223,8 +224,7 @@ def _build_schema(
 
 
 def _is_call_ctx(annotation: Any) -> bool:
+    """Return whether the annotation is the `RunContext` class, parameterized or not."""
     from .tools import RunContext
 
-    return annotation is RunContext or (
-        _typing_extra.is_generic_alias(annotation) and get_origin(annotation) is RunContext
```

Comment:
No need to check that annotation is a generic alias, it is guaranteed to be if the origin is `RunContext`